src/main/java/com/minecolonies/coremod/network/messages/OpenInventoryMessage.java

```diff
@@ -161,6 +161,8 @@ private static void doCitizenInventory(final OpenInventoryMessage message, final
             {
                 citizen.getInventoryCitizen().setCustomName(message.name);
             }
+            //TODO(OrionDevelopment): Convert next line to:
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)